modules/apps/static/osgi/osgi-util/src/testIntegration/java/com/liferay/osgi/util/test/TestInstance.java

```diff
@@ -29,12 +29,12 @@ public TrackedTwo getTrackedTwo() {
 		return _trackedTwo;
 	}
 
-	@Reference
+	@Reference(unbind = "-")
```

Comment:
@hhuijser this is not right, the Reference here is com.liferay.osgi.util.service.Reference, it is not the osgi one, it does not have the unbind file.
Can you update the SF scan to only pick up on the org.osgi.service.component.annotations.Reference?

@tinatian will help you on this pull. For now, we can just remove the apply to TestInstance, to see whether there is other failures.

@hhuijser please send your updated pull to @tinatian , thanks!